src/datashare/azext_datashare/generated/_params.py

```diff
@@ -0,0 +1,307 @@
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('account_name', help='The name of the share account.')
+        c.argument('share_name', help='The name of the share.')
+        c.argument('skip_token', help='Continuation token')
+
+    with self.argument_context('datashare invitation show') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('account_name', help='The name of the share account.')
+        c.argument('share_name', help='The name of the share.')
+        c.argument('invitation_name', help='The name of the invitation.')
+
+    with self.argument_context('datashare invitation create') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('account_name', help='The name of the share account.')
+        c.argument('share_name', help='The name of the share.')
+        c.argument('invitation_name', help='The name of the invitation.')
+        c.argument('properties_target_active_directory_id', help='The target Azure AD Id. Can\'t be combined with email.')
+        c.argument('properties_target_email', help='The email the invitation is directed to.')
+        c.argument('properties_target_object_id', help='The target user or application Id that invitation is being sent to. Must be specified along TargetActiveDirectoryId. This enables sending invitations to specific users or applications in an AD tenant.')
+
+    with self.argument_context('datashare invitation delete') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('account_name', help='The name of the share account.')
+        c.argument('share_name', help='The name of the share.')
+        c.argument('invitation_name', help='The name of the invitation.')
+
+    with self.argument_context('datashare share list') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('account_name', help='The name of the share account.')
+        c.argument('skip_token', help='Continuation token')
+
+    with self.argument_context('datashare share show') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('account_name', help='The name of the share account.')
+        c.argument('share_name', help='The name of the share.')
+
+    with self.argument_context('datashare share create') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The resource group name.')
+        c.argument('account_name', help='The name of the share account.')
+        c.argument('share_name', help='The name of the share.')
+        c.argument('properties_description', help='Share description.')

```

Comment:
the name is strange, should it be `description`? if this code gen issue? if yes pls feedback to codegen team

Reply:
updated in manual code.